### DISCUSSION TITLE
MacOS builds: legacy macOS version support

### DISCUSSION THREAD

**Discussion by Errand24** [86Box/86Box#3765] [Ideas]
Hello dear 86Box developers!

I've tried the early b3737 build on 10.9, but as expected there's no luck: the app shows message that it is damaged and asks to remove it.

I wonder: if it's possible to make builds for the older macOS versions (10.7, 10.8, 10.9)?
Is it so much effort to do it?

I would agree to help with testing on 10.7, 10.8 and 10.9.

Thanks in advance for considering!

**Comment by cold-brewed** [comment]
Hello! 

Unfortunately, our use of Qt limits how far we can go back with platform compatibility. If you look at the [Qt 5 compatibility matrix](https://doc.qt.io/qt-5/supported-platforms.html) you'll see that 10.13 is the minimum macOS version we can target. Our macOS bundles currently target 10.13 as a minimum due to this. You'll likely run into this limitation with any 86Box macOS build you try, as they have always used Qt 5.

There would be additional hurdles, regardless: potential toolchain issues along with the need for additional build infrastructure for those older toolchains.

Qt 6 is even more restrictive: [Qt 6 matrix](https://doc.qt.io/qt-6/supported-platforms.html). The lowest supported macOS there is 11. The current plan is to stay on Qt 5 as long as it is maintained, though you can optionally compile with Qt 6 today.

**Comment by dhrdlicka** [comment]
Technically there is the old SDL frontend that could be revived for this purpose, but this would likely be maintained on a "build it yourself" basis.
